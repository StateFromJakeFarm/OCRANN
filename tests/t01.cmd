  train_input1.txt train_output1.txt test_input1.txt test_output1.txt structure1.txt weights1.txt encoding.txt 0.01 300
