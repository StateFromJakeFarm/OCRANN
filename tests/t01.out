2.498235885873 0.773095160509 
0
1
0
2
2
0
1
2
2
0
2
2
0
0
0
0
1
2
2
0
1
0
2
1
1
0
2
2
2
2
1
0
1
2
0
0
0
1
0
0
1
0
0
2
2
1
2
0
0
1
0
2
2
0
0
1
1
2
0
0
2
0
1
1
2
2
2
0
1
1
0
0
1
2
0
0
0
0
2
2
0
2
1
2
0
0
1
1
0
2
2
1
0
2
0
0
2
0
0
0
1.000000000000
